### DISCUSSION TITLE
5.x版本怎么友好升级到6.x？

### DISCUSSION THREAD

**Discussion by venuszjp** [w7corp/easywechat#2692] [Q&A]
6.x版本API调用方式改变了，5.x版本怎么友好升级到6.x？

**Comment by overtrue** [comment]
找到使用的地方一处处改哈，通常来讲不是做的专门的微信服务封装，都没有太多需要改的地方。

**Reply by venuszjp** [reply]
那怎么绑定cache和刷新access_token呢
